examples/python/db_api_example.py

```diff
@@ -0,0 +1,149 @@
+import pygridgain_dbapi
+from pygridgain_dbapi import DatabaseError
+
+def create_simple_connection():
+  """Create a basic connection to GridGain cluster."""
+  addr = ['127.0.0.1:10800']
+  return pygridgain_dbapi.connect(address=addr, timeout=10)
+
+def create_ssl_connection():
+  """Create SSL-enabled connection to GridGain cluster."""
+  addr = ['127.0.0.1:10800']
+  return pygridgain_dbapi.connect(
+      address=addr,
+      timeout=10,
+      use_ssl=True,
+      ssl_keyfile='<path_to_ssl_keyfile.pem>',
+      ssl_certfile='<path_to_ssl_certfile.pem>',
+      # Optional: ssl_ca_certfile='<path_to_ssl_ca_certfile.pem>'
+  )
+
+def create_authenticated_connection():
+  """Create authenticated connection to GridGain cluster."""
+  addr = ['127.0.0.1:10800']
+  return pygridgain_dbapi.connect(
+      address=addr,
+      timeout=10,
+      identity='user',
+      secret='password'
+  )
+
+def basic_operations_example():
+  """Demonstrate basic CRUD operations."""
+  conn = create_simple_connection()
+
+  try:
+    with conn.cursor() as cursor:
+      # Drop table if exists (for clean runs)
+      cursor.execute('DROP TABLE IF EXISTS Person')
+
+      # Create table
+      cursor.execute('''
+                CREATE TABLE Person(
+                    id INT PRIMARY KEY, 
+                    name VARCHAR, 
+                    age INT
+                )
+            ''')
+
+      # Sample data
+      sample_data = [
+        [1, "John", 30],
+        [2, "Jane", 32],
+        [3, "Bob", 28]
+      ]
+
+      # Insert data (fixed table name)
+      cursor.executemany('INSERT INTO Person VALUES(?, ?, ?)', sample_data)
+
+      # Query data
+      cursor.execute('SELECT * FROM Person ORDER BY id')
+      results = cursor.fetchall()
+
+      print("All persons in database:")
+      for row in results:
+        print(f"ID: {row[0]}, Name: {row[1]}, Age: {row[2]}")
+
+      # Query specific record
+      cursor.execute('SELECT name, age FROM Person WHERE id = ?', [2])
+      person = cursor.fetchone()
+      if person:
+        print(f"\nFound person: {person[0]}, age {person[1]}")
+
+  except DatabaseError as e:
```

Comment:
Why only `DatabaseError`? There is a bunch of possible errors to catch. Let's catch general `Exception`